apps/angular/src/app/features/anime-table/anime-table.component.ts

```diff
@@ -16,19 +32,48 @@ enum ColumnKey {
 	Status = 'status',
 }
 
+const COLUMN_TO_QUERY_PARAM: Readonly<Record<string, string>> = {
+	airedStart: 'aired__startswith',
+	titleEnglish: 'title_eng',
+	status: 'status',
+};
+
 /** Anime table component. */
 @Component({
 	selector: 'camp-anime-table',
 	templateUrl: './anime-table.component.html',
 	styleUrl: './anime-table.component.css',
 	standalone: true,
-	imports: [MatTableModule, AsyncPipe, DatePipe, EmptyPipe],
+	imports: [
+		MatTableModule,
+		MatPaginatorModule,
+		DatePipe,
+		EmptyPipe,
+		MatSortModule,
+		SearchFormComponent,
+		AnimeTypeFilterComponent,
+	],
 })
-export class AnimeTableComponent {
+export class AnimeTableComponent implements OnInit, AfterViewInit {
 	private readonly animeService = inject(AnimeService);
 
-	/** Represents anime list. */
-	protected readonly anime$ = this.animeService.getAll();
+	private readonly activatedRoute = inject(ActivatedRoute);
+
+	private readonly navigationService = inject(NavigationService);
+
+	private readonly destroyReference = inject(DestroyRef);
+
+	private readonly pageNumber = signal(0);
+
+	private readonly pageSize = signal(25);
```

Comment:
Usually, we create a constant variable above component class, for example `const INITIAIL_PAGE_SIZE = 25`;
```suggestion
	private readonly pageSize = signal(INITIAIL_PAGE_SIZE);
```